### ISSUE TITLE
provide the specification for the redis URI

### ISSUE THREAD

**Issue by teto** [redis/redis-doc#2523]
I was looking for the spec expected by `redis-cli -u`  since https://github.com/informatikr/hedis doesn't interpret it the same way as redis-cli (hedis will parse password as the username in `redis://password@localhost` while redis-cli uses it as password-) but I could not find it anywhere.

I found this resource but I am not sure whether it's up to date/ https://www.iana.org/assignments/uri-schemes/prov/redis and the best one imho: 
https://github.com/lettuce-io/lettuce-core/wiki/Redis-URI-and-connection-details#uri-syntax

I wish something as fundamental were more clearly documented and cherry on the cake if you can reference doc sections with anchors:
![image](https://github.com/redis/redis-doc/assets/886074/de8fa7e5-b86e-48fa-ae56-179fdb96698a)

**Comment by dwdougherty** [comment]
Hello Matthieu. The following URL will take directly to that page and anchor:

https://redis.io/docs/ui/cli/#host-port-password-and-database

**Comment by teto** [comment]
thanks for the link but you kinda have to guess it ?  the point about the missing uri spec still exists

**Comment by dwdougherty** [comment]
I don't think I understand exactly what you're asking for, Matthieu; apologies. Maybe you're asking to make that section of the doc a third-level heading, so:

```
## Host, port, password, and database
...
### Connection string URI pattern

Some or all of this information can also be provided by using the -u <uri> option and the URI pattern redis://user:password@host:port/dbnum:
...
```

**Comment by teto** [comment]
From the browser UI, I would like to be able to generate redis.io/docs/ui/cli/#host-port-password-and-database
but here I have to guess
![image](https://github.com/redis/redis-doc/assets/886074/bb703b91-a5f3-4a64-b334-a5bc5c4ed480)

**Comment by dwdougherty** [comment]
I understand now. Apologies for being a bit slow on the uptake. :) The links on the right side of the page behave the way you want. In other words, clicking on one of those links will place the full link in the URL field in your browser. Will that not suffice for now?

**Comment by teto** [comment]
It's been a while so I dont remember but either I didn't notice that (shamne on me) or I did, tried to click on the current one and it did nothing. 
![image](https://github.com/redis/redis-doc/assets/886074/62735d9d-8a69-44ba-a9b3-498578a3cfac) 

I dont want to argue about the UI though since you've just shown me it was possible to get a link (sry I did not notice) but back to my main complaint: I was searching redis docs for a formal description of the redis uri to confirm if https://github.com/informatikr/hedis/issues/215 was a bug. 

https://www.iana.org/assignments/uri-schemes/prov/redis looks like the best one but I couldn't find it on the redis website while I think it deserves a reference.
